specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/preview/2025-07-01-preview/examples/PutModerationListItem.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "accountName": "myCogsAccount",
+    "moderationListName": "myModerationList1",
+    "moderationListItemName": "term1",
+    "api-version": "2025-07-01-preview",
+    "moderationTerm": {
```

Comment:
please rename the file to *Term.json and also put operation can create mutiple terms a moderationList